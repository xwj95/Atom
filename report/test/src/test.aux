\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\contentsline {section}{\numberline {1}引言}{2}{section.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}编写目的}{2}{subsection.1.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}测试方案}{2}{subsection.1.2}}
\@writefile{toc}{\contentsline {section}{\numberline {2}单元测试}{2}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}ALU模块}{2}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}指令解码模块}{2}{subsection.2.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.3}访存模块}{2}{subsection.2.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.4}通用寄存器模块}{2}{subsection.2.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.5}CP0模块}{2}{subsection.2.5}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.6}异常处理模块}{2}{subsection.2.6}}
\@writefile{toc}{\contentsline {section}{\numberline {3}指令片段测试}{2}{section.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.1}逻辑、移位操作与空指令的测试}{2}{subsection.3.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.2}移动操作指令的测试}{2}{subsection.3.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.3}算术操作指令的测试}{2}{subsection.3.3}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.4}转移指令的测试}{2}{subsection.3.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.5}加载存储指令的测试}{2}{subsection.3.5}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.6}协处理器访问指令的测试}{2}{subsection.3.6}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.7}异常相关指令的测试}{2}{subsection.3.7}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.8}与数据冲突有关的测试}{2}{subsection.3.8}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.9}与结构冲突有关的测试}{2}{subsection.3.9}}
\@writefile{toc}{\contentsline {subsection}{\numberline {3.10}与控制冲突有关的测试}{2}{subsection.3.10}}
\@writefile{toc}{\contentsline {section}{\numberline {4}系统测试}{2}{section.4}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}模拟操作系统测试}{2}{subsection.4.1}}
