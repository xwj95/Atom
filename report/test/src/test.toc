\contentsline {section}{\numberline {1}引言}{2}{section.1}
\contentsline {subsection}{\numberline {1.1}编写目的}{2}{subsection.1.1}
\contentsline {subsection}{\numberline {1.2}测试方案}{2}{subsection.1.2}
\contentsline {section}{\numberline {2}单元测试}{2}{section.2}
\contentsline {subsection}{\numberline {2.1}ALU模块}{2}{subsection.2.1}
\contentsline {subsection}{\numberline {2.2}指令解码模块}{2}{subsection.2.2}
\contentsline {subsection}{\numberline {2.3}访存模块}{2}{subsection.2.3}
\contentsline {subsection}{\numberline {2.4}通用寄存器模块}{2}{subsection.2.4}
\contentsline {subsection}{\numberline {2.5}CP0模块}{2}{subsection.2.5}
\contentsline {subsection}{\numberline {2.6}异常处理模块}{2}{subsection.2.6}
\contentsline {section}{\numberline {3}指令片段测试}{2}{section.3}
\contentsline {subsection}{\numberline {3.1}逻辑、移位操作与空指令的测试}{2}{subsection.3.1}
\contentsline {subsection}{\numberline {3.2}移动操作指令的测试}{2}{subsection.3.2}
\contentsline {subsection}{\numberline {3.3}算术操作指令的测试}{2}{subsection.3.3}
\contentsline {subsection}{\numberline {3.4}转移指令的测试}{2}{subsection.3.4}
\contentsline {subsection}{\numberline {3.5}加载存储指令的测试}{2}{subsection.3.5}
\contentsline {subsection}{\numberline {3.6}协处理器访问指令的测试}{2}{subsection.3.6}
\contentsline {subsection}{\numberline {3.7}异常相关指令的测试}{2}{subsection.3.7}
\contentsline {subsection}{\numberline {3.8}与数据冲突有关的测试}{2}{subsection.3.8}
\contentsline {subsection}{\numberline {3.9}与结构冲突有关的测试}{2}{subsection.3.9}
\contentsline {subsection}{\numberline {3.10}与控制冲突有关的测试}{2}{subsection.3.10}
\contentsline {section}{\numberline {4}系统测试}{2}{section.4}
\contentsline {subsection}{\numberline {4.1}模拟操作系统测试}{2}{subsection.4.1}
