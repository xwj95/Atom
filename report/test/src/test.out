\BOOKMARK [1][-]{section.1}{引言}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{编写目的}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{测试方案}{section.1}% 3
\BOOKMARK [1][-]{section.2}{单元测试}{}% 4
\BOOKMARK [2][-]{subsection.2.1}{ALU模块}{section.2}% 5
\BOOKMARK [2][-]{subsection.2.2}{指令解码模块}{section.2}% 6
\BOOKMARK [2][-]{subsection.2.3}{访存模块}{section.2}% 7
\BOOKMARK [2][-]{subsection.2.4}{通用寄存器模块}{section.2}% 8
\BOOKMARK [2][-]{subsection.2.5}{CP0模块}{section.2}% 9
\BOOKMARK [2][-]{subsection.2.6}{异常处理模块}{section.2}% 10
\BOOKMARK [1][-]{section.3}{指令片段测试}{}% 11
\BOOKMARK [2][-]{subsection.3.1}{逻辑、移位操作与空指令的测试}{section.3}% 12
\BOOKMARK [2][-]{subsection.3.2}{移动操作指令的测试}{section.3}% 13
\BOOKMARK [2][-]{subsection.3.3}{算术操作指令的测试}{section.3}% 14
\BOOKMARK [2][-]{subsection.3.4}{转移指令的测试}{section.3}% 15
\BOOKMARK [2][-]{subsection.3.5}{加载存储指令的测试}{section.3}% 16
\BOOKMARK [2][-]{subsection.3.6}{协处理器访问指令的测试}{section.3}% 17
\BOOKMARK [2][-]{subsection.3.7}{异常相关指令的测试}{section.3}% 18
\BOOKMARK [2][-]{subsection.3.8}{与数据冲突有关的测试}{section.3}% 19
\BOOKMARK [2][-]{subsection.3.9}{与结构冲突有关的测试}{section.3}% 20
\BOOKMARK [2][-]{subsection.3.10}{与控制冲突有关的测试}{section.3}% 21
\BOOKMARK [1][-]{section.4}{系统测试}{}% 22
\BOOKMARK [2][-]{subsection.4.1}{模拟操作系统测试}{section.4}% 23
